composer.json

```
@@ -25,7 +25,7 @@
         "handcraftedinthealps/elasticsearch-dsl": "^5.0.7.1 || ^6.2.0.1 || ^7.2.0.1",
         "jms/serializer": "^3.3",
         "jms/serializer-bundle": "^3.3 || ^4.0",
-        "sulu/sulu": "~2.4.13 || ^2.5.9@dev",
+        "sulu/sulu": "2.6.x-dev",
```

We should keep it copmatible to 2.5 atleast. I can have a look at it. Do you know whats breaking there @martinlagler ?

```suggestion
        "sulu/sulu": "~2.4.13 || ^2.5.9@dev",
```